CMakeLists.txt

```diff
@@ -667,7 +667,10 @@ file(STRINGS util/go_tests.txt GO_TESTS)
 set_property(DIRECTORY APPEND PROPERTY CMAKE_CONFIGURE_DEPENDS
              util/go_tests.txt)
 
-add_custom_target(
+# airtime: start
+if (NOT ANDROID)
+  add_custom_target(
```

Comment:
I'm confused. The opening parenthesis is here in airtime tag but what about the closing parenthesis?

Reply:
this is indeed  confusing, I took the block present in develop and intention was not to run these tests in android. i think it should have been like below

> # airtime: start
> if (NOT ANDROID)
> # airtime: end
> 
>   add_custom_target(
>     run_tests
>     COMMAND ${GO_EXECUTABLE} test ${GO_TESTS}
>     COMMAND ${GO_EXECUTABLE} run util/all_tests.go -build-dir
>             ${CMAKE_CURRENT_BINARY_DIR}
>     COMMAND cd ssl/test/runner &&
>             ${GO_EXECUTABLE} test -shim-path $<TARGET_FILE:bssl_shim>
>               ${HANDSHAKER_ARGS} ${RUNNER_ARGS}
>     WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
>     DEPENDS all_tests bssl_shim handshaker fips_specific_tests_if_any
>     USES_TERMINAL)
> # airtime: start
> endif()  
> # airtime: end
>